src/Observable.ts

```diff
@@ -21,7 +21,7 @@ import {rxSubscriber} from'./symbol/rxSubscriber';
 export class Observable<T> implements CoreOperators<T>  {
   source: Observable<any>;
   operator: Operator<any, T>;
-  _isScalar: boolean = false;
+  _isScalar = false;
```

Comment:
I would like this to be discussed as separate. Lot of analysis tools or editor recommends this pattern (to not explicitly specify type if it's obvious by assigned value) but it's currently common in our codebases. Once we decide which way to go, single PR change it once would works.